Makefile

```diff
@@ -25,6 +25,12 @@ build:
 bin:
 	GOOS=linux $(GO) build -o $(BINDIR)/$(OPERATOR_NAME) $(GOFLAGS) -ldflags '$(LDFLAGS)' ./cmd/manager
 
+.PHONY: bundle
+bundle:
+	docker build -t $(BUNDLE_REPO)/$(BUNDLE_IMG_NAME):tmp -f build/Dockerfile .
+	docker push $(BUNDLE_REPO)/$(BUNDLE_IMG_NAME):tmp
```

Comment:
It's necessary to push the image because the tool opm cannot load local images :).